experimental/dds/tree-graphql/package.json

```diff
@@ -58,8 +59,14 @@
     "eslint": "~8.6.0",
     "mocha": "^10.0.0",
     "nyc": "^15.0.0",
-    "prettier": "^2.3.1",
+    "prettier": "~2.3.1",
```

Comment:
Just curious why this was necessary? (Besides "we added the rule in the root package.json")

Reply:
The build-tools upgrade includes new policy-checks, including one that ensures some deps use `~` instead of `^` (because they introduce changes we consider breaking in minor versions).